nautobot/utilities/testing/filters.py

```diff
@@ -36,12 +36,15 @@ def get_filterset_test_values(self, field_name, queryset=None):
             qs_count = queryset.count()
             values_with_count = queryset.values(field_name).annotate(count=Count(field_name)).order_by("count")
             for value in values_with_count:
+                # skip null values
+                if value[field_name] is None:
+                    continue
```

Comment:
Would it be possible and more efficient to exclude the null values from the queryset upfront rather than including them in the query and having to skip them individually?